pynamodb/connection/base.py

```diff
@@ -578,10 +578,9 @@ def get_meta_table(self, table_name: str) -> MetaTable:
         """
         Returns information about the table's schema.
         """
-        try:
-            return self._tables[table_name]
-        except KeyError:
-            raise TableError(f"Meta-table for '{table_name}' not initialized") from None
+        if table_name not in self._tables:
+            self.describe_table(table_name)
```

Comment:
@ikonst Don't you want to do add_meta_table here after the describe call ?

Reply:
Oh ignore. This is self.describe_table. 

Looks good,

Reply:
Yeah, `describe_table` populates the `_tables` with what it learns.